src/main/java/com/zendesk/maxwell/schema/columndef/DateTimeColumnDef.java

```diff
@@ -44,6 +46,18 @@ else if ( value instanceof Timestamp )
 			return "";
 	}
 
+	private String formatLong(Long value) {
+		final int second = (int)(value % 100); value /= 100;
+		final int minute = (int)(value % 100); value /= 100;
+		final int hour = (int)(value % 100); value /= 100;
+		final int day = (int)(value % 100); value /= 100;
+		final int month = (int)(value % 100);
+		final int year = (int)(value / 100);
```

Comment:
Hm. Is the open replicator doing this encoding underneath? "2014-01-02 12:40:56" => 20140102124056